modules/_node-scripts/configuration/getProjectMain.mjs

```diff
@@ -0,0 +1,10 @@
+import require from '../util/require.mjs';
+
+/**
+ * @returns the proejct relative path of the main entry point
+ */
+export default function getProjectExports() {
+	const {main} = require('./node-scripts.config.js');
+
+	return main;
+}
```

Comment:
these "require then return" utils are so odd to me. I'm sure it has to do with the require util you have, but it seems like boilerplate overkill

Reply:
The reason why I use require is that it caches the node-scripts.config.js file in memory. That way we can keep different functions for different unrelated config values and, at the same time, not hurt build's performance because we read the same file over and over again.